roadmap-site/README.md

```diff
@@ -0,0 +1,95 @@
+# Challenges
+
+## User Object
+
+
+same as the users object on Users Api
+- [Users](/users)
```

Comment:
How about `[User object](/users#user-object)`?
Wouldn't that make more sense?

Reply:
Are we sending everything in user object? Why?
Should we not send only the things required, like name, profile pic, and id? Why send additional details?

Reply:
For now send the complete user object as its defined on the users API .. I might need some additional user info .. so for now lets stick with it.. when things progress.. I will update it accordingly if at all I feel there are certain information which I do not need

Reply:
Sure.